docs/bin/plugin_formatter.py

```diff
@@ -125,12 +125,13 @@ def write_data(text, options, outputname, module):
 #####################################################################################
 
 
-def list_modules(module_dir, depth=0):
+def list_modules(module_dir, depth=0, limit_to_modules="all"):
```

Comment:
I'll see what I can do to switch to lists and conditionals. One problem is my `Makefile` expertise is virtually non-existent (and don't have much time/will to learn it atm :)